diff --git "a/.\\mod.ini.old" "b/.\\mod.ini"
index 91258f2..a1b2098 100644
--- "a/.\\mod.ini.old"
+++ "b/.\\mod.ini"
@@ -16,7 +16,7 @@ global persist $switch = 0
 
 [KeySwitch]
 condition = $object_detected == 1
-key = \
+key = alt 1
 type = cycle
 $switch = 0,1
 
@@ -123,9 +123,9 @@ endif
 vb0 = ref ResourceBypassVB0
 
 [TextureOverrideComponent0]
-hash = d53ea67e
+hash = f9935b48
 match_first_index = 0
-match_index_count = 43863
+match_index_count = 44688
 $object_detected = 1
 if $mod_enabled
 	local $state_id_0
@@ -159,7 +159,7 @@ endif
 filename = Textures/Components-0 t=146320ad.dds
 
 [TextureOverrideTexture0]
-hash = 146320ad
+hash = 9fa0ecef
 match_priority = 0
 this = ResourceTexture0
 
